docs/reference/modules/cluster/allocation_awareness.asciidoc

```diff
@@ -21,7 +21,7 @@ attribute called `rack_id` -- we could use any attribute name.  For example:
 
 [source,sh]
 ----------------------
-./bin/elasticsearch --node.rack_id rack_one <1>
+./bin/elasticsearch -Ees.node.rack_id=rack_one <1>
```

Comment:
I'm fine with that, but I think that removing this should be addressed in the [same followup PR](https://github.com/elastic/elasticsearch/pull/17088/files#r55955307) that removes allowing setting settings via system properties. The reason is because setting settings via system properties *should* continue to require the `es.` prefix. I think the followup should just remove the ability to set settings via system properties, and completely do away with the `es.` prefix handling then.